seabird.proto

```diff
@@ -166,6 +167,14 @@ message StreamInfoResponse {
 
   map<string, CommandMetadata> commands = 3;
 
-  // TODO: find a way to track IPs related to plugins - maybe deny connections
-  // from an IP other than the registration
+  string ip = 4;
```

Comment:
How are we handling ipv4 vs ipv6? Would this be better as remote_addr, or do you think IP is fine?

Reply:
Fair point. My intent was to track either IPv4 or IPv6 here as this is a generic `string`. I like `remote_address`!